packages/time-series/lib/commands/INFO_DEBUG.ts

```diff
@@ -1,57 +1,57 @@
-// import {
-//     transformArguments as transformInfoArguments,
-//     InfoRawReply,
-//     InfoReply,
-//     transformReply as transformInfoReply
-// } from './INFO';
+import { BlobStringReply, Command, NumberReply, SimpleStringReply } from "@redis/client/dist/lib/RESP/types";
+import INFO, { InfoRawReply, InfoReply } from "./INFO";
 
-// export { IS_READ_ONLY, FIRST_KEY_INDEX } from './INFO';
+type InfoDebugRawReply = [
+  ...InfoRawReply,
+  'keySelfName',
```

Comment:
same comment re TuplesToMapReply (but perhaps more complicated due to the ... expansion)